modules/release-notes/ref-release-notes-compatibility-matrix.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: REFERENCE
+[id="compatibility-matrix"]
+= Compatibility matrix
+
+The following table lists products and their versions compatible with {product-short} {product-version}:
+
+.Platform Compatibility Matrix
+[cols="1,1,1,1,1,1,1",options="header"]
+|===
+| {product-short} Version | {ocp-brand-name} | {eks-brand-name} ({eks-short}) | {aks-brand-name} ({aks-short}) | {gcp-brand-name} ({gcp-short}) | {gke-brand-name} ({gke-short}) | Backstage Versions
+
+| {product-version}
```

Comment:
To maintain consistency with the 1.3 versions, I would suggest adding code quotes to all version attributes for 1.4 as well.
```suggestion
| `{product-version}`
```